src/main/php/PHPMD/TextUI/XdebugOptionHandler.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace PHPMD\TextUI;
+
+use Composer\XdebugHandler\XdebugHandler;
+
+/**
+ * Xdebug CLI Option Handler
+ *
+ * Enables instead of disables Xdebug, if called with "--xdebug" CLI option.
+ */
+class XdebugOptionHandler extends XdebugHandler
+{
+    /**
+     * Rebuilds the run command with Xdebug enabled, instead, if CLI option "--xdebug" is used
```

Comment:
This will help phpstan to know precisely what is the type of this array and so not to assume the key can be string:
```suggestion
     * Rebuilds the run command with Xdebug enabled, instead, if CLI option "--xdebug" is used.
     *
     * @param list<string> $command
```